draft-ietf-quic-transport.md

```diff
@@ -3192,8 +3192,8 @@ packets at the same encryption level.
 
 Receivers MAY route based on the information in the first packet contained in a
 UDP datagram.  Senders MUST NOT coalesce QUIC packets for different connections
-into a single UDP datagram.  Receivers SHOULD ignore any subsequent packets with
-a different Destination Connection ID than the first packet in the datagram.
+into a single UDP datagram.  Receivers SHOULD ignore any subsequent packets for
+a different connection than the first packet in the datagram.
 
```

Comment:
I do not agree with this change. It complicates packet parsing, for very little practical gain.

Reply:
I do not agree with this change. It's unnecessary complexity for no clear benefit. We should fix the sender text to say MUST NOT coalesce different connection IDs into a single datagram. If there are scenarios where this is needed let's revisit in QUIC v2.